src/panels/lovelace/strategies/areas/editor/hui-areas-dashboard-strategy-editor.ts

```diff
@@ -0,0 +1,104 @@
+import { mdiTextureBox } from "@mdi/js";
+import { css, html, LitElement, nothing } from "lit";
+import { customElement, property, state } from "lit/decorators";
+import { fireEvent } from "../../../../../common/dom/fire_event";
+import { getAreaContext } from "../../../../../common/entity/context/get_area_context";
+import "../../../../../components/ha-expansion-panel";
+import type { HomeAssistant } from "../../../../../types";
+import type { LovelaceStrategyEditor } from "../../types";
+import type { AreasDashboardStrategyConfig } from "../areas-dashboard-strategy";
+import { getAreas } from "../helpers/areas-strategy-helpers";
+import "./ha-items-visibility-editor";
+import type {
+  VisibilityEditorItem,
+  VisibilityEditorValue,
+} from "./ha-items-visibility-editor";
+
+@customElement("hui-areas-dashboard-strategy-editor")
+export class HuiAreasDashboardStrategyEditor
+  extends LitElement
+  implements LovelaceStrategyEditor
+{
+  @property({ attribute: false }) public hass?: HomeAssistant;
+
+  @state()
+  private _config?: AreasDashboardStrategyConfig;
+
+  public setConfig(config: AreasDashboardStrategyConfig): void {
+    this._config = config;
+  }
+
+  protected render() {
+    if (!this.hass || !this._config) {
+      return nothing;
+    }
+
+    const areas = getAreas(this.hass.areas, [], this._config.areas_order);
+
+    const items: VisibilityEditorItem[] = areas.map((area) => {
+      const { floor } = getAreaContext(area.area_id, this.hass!);
+      return {
+        value: area.area_id,
+        label: area.name,
+        icon: area.icon ?? undefined,
+        iconPath: mdiTextureBox,
+        description: floor?.name,
+      };
+    });
+
+    const value = {
+      order: this._config.areas_order ?? [],
+      hidden: this._config.hidden_areas ?? [],
+    };
+
+    return html`
+      <ha-expansion-panel leftChevron .expanded=${true} outlined>
+        <h3 slot="header">
```

Comment:
Why do we use a `h3` here? Shouldnt `ha-expansion-panel` be responsible for that?